src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -26,6 +30,27 @@ def _convert_well_trajectory(self) -> ProfileDescription:
         x, y = self.score_input.read_well_trajectory()
         return ProfileDescription(x_and_y=XAndYDescription(x=x, y=y))
 
+    def convert_materials(self) -> List[MaterialDescription]:
+        """Convert list of materials from SCORE file"""
+        material_descriptions = []
+        material_list = (
+            [self.score_input.read_cement_material()]
+            + self.score_input.read_tubing_materials()
+            + self.score_input.read_lithology_materials()
+        )
+        for data in material_list:
+            material_descriptions.append(
+                MaterialDescription(
```

Comment:
Didi you try to create a `MaterialDescription` using the dict as it?
For example:
```
material_descriptions.append(MaterialDescription(**data))
```

Reply:
You can use in dict the very same argument names as keys.

Reply:
I thought about it after define the dictionary keys, they have different names so I gave up. rsrs

Reply:
But you defined the keys, why not define with the same name?
Code will be much more clean.

Reply:
The data comes with `young_modulus` and `poisson_ratio` so I couldn't do like this.